src/test.js

```diff
@@ -480,15 +486,42 @@ Test.prototype = {
 };
 
 export function pushFailure() {
+	return internalPushFailure.apply(
+		null,
+		Array.prototype.concat.apply( [ false ], arguments )
+	);
+}
+
+export function pushFailureAndResume() {
+	return internalPushFailure.apply(
+		null,
+		Array.prototype.concat.apply( [ true ], arguments )
+	);
+}
+
+function internalPushFailure( shouldResume ) {
 	if ( !config.current ) {
-		throw new Error( "pushFailure() assertion outside test context, in " +
-			sourceFromStacktrace( 2 ) );
+		throw new Error(
+			[
+				shouldResume ? "pushFailureAndResume()" : "pushFailure()",
+				"assertion outside test context, in",
+				sourceFromStacktrace( 2 )
```

Comment:
@gibson042 Could I get some insight on how I can find the correct number? Is the number supposed to be how many levels in the call stack get popped off (from src/core/stacktrace.js etc.) before returning the stack trace as a string?